ersilia/cli/commands/catalog.py

```diff
@@ -29,7 +31,23 @@ def catalog_cmd():
         default=False,
         help="Show more information than just the EOS identifier",
     )
-    def catalog(local=False, file_name=None, browser=False, more=False):
+    @click.option(
+        "--card",  # New --card option
```

Comment:
`--card` should be a flag with default value as False, so instead of setting its type you want to use `is_flag=True`, and `default=False`.

Reply:
In addition to that, we should expect model as 
`@click.argument("model", type=click.STRING, help=....)`